moztelemetry/stats.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import division
+
+import math
+from collections import namedtuple
+
+
+def rank(sample):
```

Comment:
It will return doubles since I'm importing division from the `__future__`, which will be more Python 3 if this project ever moves to it. W/o that import you're correct (and living in the past).

Reply:
heh, good point! I've gotta switch my local default to 3.6...